src/Splat/ServiceLocation/ModernDependencyResolver.cs

```diff
@@ -125,7 +122,7 @@ public object GetService(Type serviceType, string? contract = null)
             }
 
             var ret = _registry[pair].LastOrDefault();
-            return (ret is null ? null : ret())!;
+            return (ret != null ? ret() : null)!;
```

Comment:
`ret is not null` is the inverse of `ret is null`

`is null` and `is not null`.

Reply:
Reverted, I had played with the code and forgot how it previously was.